packages/sit-onyx/src/composables/useTimer.ts

```diff
@@ -0,0 +1,68 @@
+import { computed, ref, watch, type Ref } from "vue";
+
+type TimerOptions = {
+  endTime: Ref<string> /** expect ISO string @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString} */;
+  isPaused?: Ref<boolean>;
+  startImmediately?: boolean;
```

Comment:
As already commented, I don't think we need this :) Imho the timer should only depend on the endTime so we don't have a difference between the displayed tie and the passend endTime value

Reply:
you want me to remove `startImmediately ` as well ?

Reply:
removing it